adapters/vm2core/vm2core.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func AdaptExecutionResources(resources *vm.ExecutionResources) *core.ExecutionResources {
-	return &core.ExecutionResources{
+	return &core.ExecutionResources{ //nolint:exhaustruct
```

Comment:
is there a field that missing?

Reply:
Yeah TotalGasConsumed. I wasn't sure how to populate it